commands/publish.js

```diff
@@ -1,29 +1,27 @@
-'use strict';
-
-const { join } = require('path');
-const fetch = require('node-fetch');
-const ora = require('ora');
-const chalk = require('chalk');
-const {
-    helpers: { configStore },
-} = require('@eik/common');
-const PublishPackage = require('../classes/publish/package/index');
-const {
+import { join } from 'path';
+import fetch from 'node-fetch';
```

Comment:
We could use this opportunity to drop `node-fetch` as a dependency and use the built-in. Would mean a mininum of Node 18, but that's fine IMO.

Reply:
Removed in latest commit